qiskit/circuit/instruction.py

```diff
@@ -277,3 +277,42 @@ def qasm(self):
                 [str(i) for i in self.params]))
 
         return self._qasmif(name_param)
+
+    def argument_expansion(self, qargs, cargs):
+        """
+        Validation and handling of the arguments and its relationship. For example:
+        `cx([q[0],q[1]], q[2])` means `cx(q[0], q[2]); cx(q[1], q[2])`. This method
+        yields the arguments in the right grouping. In the example:
+           in: [[q[0],q[1]], q[2]],[]
+         outs: [q[0], q[2]], []
+               [q[1], q[2]], []
+
```

Comment:
Can you add documentation around which broadcasting rules we support?

Reply:
added in 96926492